<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>581</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="backgroundLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="File">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="Open"/>
    <addaction name="Save"/>
   </widget>
   <widget class="QMenu" name="Edit">
    <property name="title">
     <string>Редактор</string>
    </property>
    <addaction name="Search"/>
    <addaction name="Replace"/>
    <addaction name="Return"/>
    <addaction name="Clear"/>
   </widget>
   <widget class="QMenu" name="FontMenu">
    <property name="title">
     <string>Шрифт</string>
    </property>
    <addaction name="actionChangeFont"/>
   </widget>
   <widget class="QMenu" name="ColorMenu">
    <property name="title">
     <string>Цвет</string>
    </property>
    <addaction name="actionChangeColor"/>
   </widget>
   <widget class="QMenu" name="menuTables">
    <property name="title">
     <string>Таблица</string>
    </property>
    <addaction name="Create"/>
   </widget>
   <addaction name="File"/>
   <addaction name="Edit"/>
   <addaction name="FontMenu"/>
   <addaction name="ColorMenu"/>
   <addaction name="menuTables"/>
  </widget>
  <action name="Open">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="Save">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="Add">
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="Search">
   <property name="text">
    <string>Поиск</string>
   </property>
  </action>
  <action name="Replace">
   <property name="text">
    <string>Замена</string>
   </property>
  </action>
  <action name="Return">
   <property name="text">
    <string>Возврат</string>
   </property>
  </action>
  <action name="Exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="Clear">
   <property name="text">
    <string>Очистка</string>
   </property>
  </action>
  <action name="actionChangeFont">
   <property name="text">
    <string>Изменить</string>
   </property>
  </action>
  <action name="actionChangeColor">
   <property name="text">
    <string>Изменить</string>
   </property>
  </action>
  <action name="Create">
   <property name="text">
    <string>Создать</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
